pootle/settings/50-project.conf

```diff
@@ -52,6 +50,7 @@ TEMPLATES = [
                 'pootle_misc.context_processors.pootle_context',
             ],
             'loaders': [
+                # Put here to accept template customization
```

Comment:
I would put it besides the loader it is referring to (perhaps it would be necessary to shorten the comment), and add an ending period.